samples/image-metadata/src/components/FocalPointDialog/index.js

```diff
@@ -0,0 +1 @@
+export { FocalPointDialog } from './FocalPointDialog';
```

Comment:
Why have an index.js?

Reply:
to simplify import statements from 
````import { FocalPointDialog } from 'components/FocalPointDialog/FocalPointDialog'````

 to 
````import { FocalPointDialog } from 'components/FocalPointDialog'````

Also in theory the index could export more things of the same `FocalPointDialog` (e.g. the styles or any other sort of config). 

As far as I've seen so far we have no fixed coding practice regarding module structure and the use of index files.